.deploy.yaml

```diff
@@ -1,5 +1,5 @@
-buildscript: unpfayep/sideloader/build.sh
-postinstall: unpfayep/sideloader/postinstall.sh
+buildscript: sideloader/build.sh
```

Comment:
@smn this also needs to be updated in the scaffold (since you've removed the 2 levels deep problem)